docs/_book.yaml

```diff
@@ -182,14 +182,20 @@ upper_tabs:
       - heading: "Google hardware"
       - title: "Getting started with QCS"
         path: /cirq/tutorials/google/start
-      - title: "Floquet calibration"
-        path: /cirq/tutorials/google/floquet
```

Comment:
I just tweeted a link to this page to go along with today's Nature publication. In general, we should be careful about URL backwards compatibility. Would it be possible to give the new "Floquet calibration: Example and benchmark" the `/floquet` URL? Does docsite have the ability to alias pages from multiple endpoints? @lamberta

Reply:
Yes, maintaining URL stability is important, particularly when SEO kicks in and you want to move pages.
You can't alias a page to two separate URLs, but you can add a redirect to point to the new page. Add an entry in the `_redirects.yaml` file located in the root of your docsite server directory.

Reply:
Thanks for the help!

Reply:
Indeed thanks! I'll add a redirect, and, more importantly, I'll like the tweet :)

Reply:
@rmlarose Did you add the redirect already? I can't see any changes to `docs/_redirects.yaml` (assuming that's where it's supposed to go).